src/OrbitClient.php

```diff
@@ -201,6 +201,8 @@ private function getRequestHeaders(array $options)
         }
         if ($this->options['env'] === 'stage') {
             $headers['x-forge-environment'] = 'stage';
+        }
+        if ($this->options['env'] !== 'live') {
```

Comment:
I think i'd rather be explicit about which environments this applies. Can maybe use in_array in php for that?

Reply:
Sounds good! I'll sort it now